src/stores/utils.ts

```diff
@@ -65,9 +68,9 @@ export const tokenInfo = (
   color: number
 ): Promise<[string, string, string]> => {
   return Promise.all([
-    tokenValue(web3, token, 'symbol'),
+    tokenValue(web3, token, 'symbol') || token.options.address.substring(0, 5),
```

Comment:
Maybe it's better to get a substring starting from 2 (as the first 2 symbols are always `0x`)?

Reply:
often times symbol is written near the balance which is numeric value. Having two decimals written one after another will look confusing. I'm not even sure 0x is not confusing as well :)

Reply:
`0x` would still be kinda confusing I think. Than maybe use color (the same as with name), smth like `TKN${color}`? Or if you want to stick with address, use only letters smth like `token.options.address.replace(/[^a-fA-F]/gi,'').substring(0,4)`?

Reply:
How about `COLOR_N`?